### ISSUE TITLE
Uncaught TypeError: Cannot read property 'toLowerCase' of undefined

### ISSUE THREAD

**Issue by bigos** [sjlevine/atom-slime#7]
[Enter steps to reproduce below:]

1. experiment with lisp-paredit
2. after a while the problem will be triggered

**Atom Version**: 1.1.0
**System**: Mac OS X 10.11.1
**Thrown From**: [atom-slime](https://github.com/sjlevine/atom-slime) package, v1.7.2

### Stack Trace

Uncaught TypeError: Cannot read property 'toLowerCase' of undefined

```
At /Users/jacekpodkanski/.atom/packages/atom-slime/lib/slime-functions.coffee:10

TypeError: Cannot read property 'toLowerCase' of undefined
  at Object.module.exports.getPackage (/Users/jacekpodkanski/.atom/packages/atom-slime/lib/slime-functions.coffee:10:36)
  at AtomSlimeEditor.module.exports.AtomSlimeEditor.stoppedEditingCallback (/Users/jacekpodkanski/.atom/packages/atom-slime/lib/atom-slime-editor.coffee:27:18)
  at /Users/jacekpodkanski/.atom/packages/atom-slime/lib/atom-slime-editor.coffee:15:44
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:86:11)
  at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1276:25

```

### Commands

```
  6x -1:00.9.0 lisp-paredit:barf-backwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  3x -0:53.7.0 lisp-paredit:slurp-backwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  2x -0:50.1.0 lisp-paredit:barf-backwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  2x -0:45.6.0 lisp-paredit:barf-forwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  2x -0:42.7.0 lisp-paredit:slurp-forwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  2x -0:40 lisp-paredit:barf-forwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  3x -0:37.9.0 lisp-paredit:slurp-forwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  2x -0:20.5.0 lisp-paredit:barf-forwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  2x -0:18.1.0 lisp-paredit:slurp-forwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  3x -0:15.2.0 lisp-paredit:barf-forwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  2x -0:12.3.0 lisp-paredit:slurp-forwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
     -0:07.4.0 lisp-paredit:barf-forwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  2x -0:06.5.0 lisp-paredit:slurp-forwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  2x -0:05.3.0 lisp-paredit:barf-forwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
     -0:03.9.0 lisp-paredit:slurp-forwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
  2x -0:01.8.0 lisp-paredit:slurp-backwards (atom-text-editor.editor.lisp-paredit-strict.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-slime, v1.7.2
language-lisp, v0.1.2
lisp-paredit, v0.5.1
minimap, v4.16.0

# Dev
No dev packages
```

**Comment by kitling** [comment]
Something that will reproduce this every time (for me):
Typing: "(()()" on your keyboard, which will produce "(()())" and this error.

**Comment by sjlevine** [comment]
Hello, thank you both for reporting this error. Your  method @kitling of reproducing reliably was very useful in debugging. This should be fixed as of 606aab7b13381ecc9f6e94f1cfbf9a95be5d433f, version 2.0.1 of this plugin.

On a side note, I've been working on adding more features and documentation to this package - please see the readme. Please let me know if you have any more problems.

Cheers,
Steve
